Automattic/jetpack
# My Jetpack: Add `ConnectScreen` footer

Closes #22649 

#### Changes proposed in this Pull Request:

* Add `footer` props on `ConnectScreen`.
* Include `footer` component on `ConnectScreen` usage in `My Jetpack`.

#### Jetpack product discussion


N/A.

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Disconnect your WordPress.com user
* Goes to `My Jetpack` page.
* Navigate to `/connection` route.
* You should see the connection footer (Text with WordPress, Google and Apple logos)

#### Demo

![Screen Shot 2022-03-03 at 15 18 02](https://user-images.githubusercontent.com/1663717/156627098-7d3df2fd-a3f4-42fe-87ad-947bae128e09.png)
